src/lib/mongoBackend/CMakeLists.txt

```diff
@@ -52,6 +52,7 @@ SET (SOURCES
     compoundResponses.cpp
```

Comment:
CNR entry missing. Something like this could be fine:

```
- Add: transient entities functionality (new virtual attribute: dateExpired) (#3000)
```

Reply:
Fixed in 5a4703d